permissions/Phone.permission

```diff
@@ -6,46 +6,28 @@
 # x-sailjail-translation-key-long-description = permission-la-phone_description
 # x-sailjail-long-description = Make phone calls
 
-# BEG systembus-org.ofono.resource
+# Ofono to place calls etc.
 dbus-system.talk org.ofono
 dbus-system.broadcast org.ofono=org.ofono.*@/*
 dbus-system.broadcast org.ofono=org.nemomobile.ofono.*@/*
-# END systembus-org.ofono.resource
 
 # Voicecall service
 dbus-user.talk org.nemomobile.voicecall
 dbus-user.broadcast org.nemomobile.voicecall=org.nemomobile.voicecall.*@/*
-
-# BEG sessionbus-org.freedesktop.Telepathy.resource
 dbus-user.talk org.freedesktop.Telepathy
 dbus-user.broadcast org.freedesktop.Telepathy=org.freedesktop.Telepathy.*@/*
 dbus-user.talk org.freedesktop.Telepathy.*
 dbus-user.broadcast org.freedesktop.Telepathy.*=org.freedesktop.Telepathy.*@/*
-# END sessionbus-org.freedesktop.Telepathy.resource
-
-mkdir     ${HOME}/.local/share/commhistory
-whitelist ${HOME}/.local/share/commhistory
 
-# BEG sessionbus-org.nemomobile.CommHistory.resource
-dbus-user.talk org.nemomobile.CommHistory
-dbus-user.broadcast org.nemomobile.CommHistory=org.nemomobile.CommHistory.*@/*
-# END sessionbus-org.nemomobile.CommHistory.resource
-
-# BEG systembus-org.ofono.SmartMessagingAgent.resource
+# ???
```

Comment:
Why is SmartMessagingAgent here?

Reply:
The process was something like: On dbus errors/complaints about a service -> service.resource file was created. These were then included in application profiles until errors went away. When all apps were gotten to working state, the resulting resource files were bundled up as feature.permission files, while trying to keep number of "features" relatively low -> done en masse -> feel free to assume that there are logical errors.

Reply:
Ok, well... I removed that and let's see what happens. libofono seems to suggest more contacts related reason for having this.